src/components/backdrop/backdrop.js

```diff
@@ -37,8 +37,11 @@ class Backdrop {
             parent.appendChild(backdropImage);
 
             if (!enableAnimation()) {
-                if (existingBackdropImage?.parentNode) {
-                    existingBackdropImage.parentNode.removeChild(existingBackdropImage);
+                if (existingBackdropImage) {
+                    const backdropContainer = getBackdropContainer();
+                    while (backdropContainer.childNodes.length > 1) {
+                        backdropContainer.removeChild(backdropContainer.firstChild);
```

Comment:
It looks like another alternative would be to set `textContent = ''`. I suspect this would be more performant than iterating over children.

Reply:
I have been busy, I will try to do this this week and see if it works.

Reply:
I agree @Kaysera, perhaps we can use this to not add more backdrops if the tab is not visible using document.visibilityState === 'hidden':


`
function onRotationInterval() {
    if (playbackManager.isPlayingLocally(['Video']) || document.visibilityState === 'hidden') {
        return;
    }

    let newIndex = currentRotationIndex + 1;
    if (newIndex >= currentRotatingImages.length) {
        newIndex = 0;
    }

    currentRotationIndex = newIndex;
    setBackdropImage(currentRotatingImages[newIndex]);
}
`